assets/src/dashboard/animations/constants.js

```diff
@@ -32,7 +32,7 @@ export const ANIMATION_EFFECTS = {
   FADE_IN: 'effect-fade-in',
   FLY_IN: 'effect-fly-in',
   PULSE: 'effect-pulse',
-  TWIRL_IN: 'twirl-in',
+  TWIRL_IN: 'effect-twirl-in',
```

Comment:
Keeping the effect names consistently prefixed with `effect-` just so we don't accidentally run into collision with our animation parts.